src/main/java/com/northcoders/makemydayapi/controller/PlacesController.java

```diff
@@ -27,4 +28,18 @@ public ResponseEntity<List<Place>> getPlaces(@PathVariable String type) throws E
         return new ResponseEntity<>(placesService.getPlaces(type), HttpStatus.OK);
     }
 
+
+
+    @GetMapping("")
+    public ResponseEntity<List<Place>> getAllPlacesByTypes(@RequestParam("types") List<String> placesTypeList) throws Exception {
```

Comment:
It would be better to name this as `type`, so user is going to combine:

`/api/v1/places?type=A&type=B`
instead of
`/api/v1/places?types=A&types=B`